packages/python-packages/apiview-copilot/prompts/mention/summarize_github_actions.prompty

```diff
@@ -0,0 +1,51 @@
+---
+name: Summarize APIView Github Action
+description: A prompt that summarizes the results of an @mention request involving Github.
+authors: 
+  - tjprescott
+version: 1.0.0
+inputs:
+  results:
+    description: The results of the @mention request to summarize.
+    type: object
+model:
+  api: chat
+  configuration:
+    type: azure_openai
+    azure_endpoint: ${env:OPENAI_ENDPOINT}
+    azure_deployment: gpt-4.1
+    api_version: 2025-03-01-preview
+  parameters:
+    temperature: 0.0
+    top_p: 1.0
+    stop: []
+    max_tokens: 16384
+sample:
+  results: | 
+    {
+      "url": "https://api.github.com/repos/tjprescott/azure-sdk-tools/issues/6",
+      "repository_url": "https://api.github.com/repos/tjprescott/azure-sdk-tools",
+      "title": "[Python APIView] Literal values are not quoted in type annotations",
+      "created_at": "2025-10-15T19:36:45Z",
+      "body": "**Bad code snippet:**\n```python\nfoo: Literal[Foo]\n```\n\n**Description:**\nThe APIView parser is incorrectly displaying literal values in type annotations without quotes. According to Python typing conventions, literal values should be quoted, e.g., `Literal[\"Foo\"]`.\n\n**Expected (good) code snippet:**\n```python\nfoo: Literal[\"Foo\"]\n```\n\nPlease update the parser to ensure literal values are properly quoted in the APIView output.",
+    }
+---
+system:
+You are an AI that summarizes the results of an @mention request from APIView. This request required taking some action on Github.
+
+# INSTRUCTIONS
+
+## NO FAILURES
```

Comment:
Summary also need to understand whether a _new_ issue was created or an existing issue was found for this. Need an eval case for this as well.